### PR TITLE
Fix bugs for gptq exporting with static_groups

### PR DESCRIPTION
**Pull request by YIYANGCAI**
## Type of Change

bug fix

## Description

Fix bugs for gptq exporting with static_groups

### CODE REVIEW

File: neural_compressor/adaptor/torch_utils/gptq.py [hunk 1]
```diff
@@ -831,6 +831,9 @@ def fasterquant(self, W, blocksize=128, percdamp=0.01, groupsize=-1, act_order=F
                         if act_order:
                             idx = perm[idx]
                         self.quantizer = groups[idx // groupsize]
+                        if (i1 + i) % groupsize == 0:
```
Comment (hshen14) [inline neural_compressor/adaptor/torch_utils/gptq.py:834]: do we have UT to cover?

Comment (YIYANGCAI) [reply neural_compressor/adaptor/torch_utils/gptq.py:834]: Of course. I have solved the exporting problem, and I will add UTs a.s.a.p.

**Comment by chensuyue** [comment]
@xin3he this change should be migrate into 3x?

**Comment by xin3he** [comment]
> @xin3he this change should be migrate into 3x?

Yes, I will sync with yiyang.

**Review by xin3he — APPROVED** [review]

**Pull request merged**